scheduler/printers.c

```diff
@@ -4027,7 +4027,7 @@ load_ppd(cupsd_printer_t *p)		/* I - Printer */
     ippAddStrings(p->ppd_attrs, IPP_TAG_PRINTER, IPP_CONST_TAG(IPP_TAG_KEYWORD), "job-creation-attributes-supported", sizeof(job_creation_print) / sizeof(job_creation_print[0]), NULL, job_creation_print);
   }
 
-  if ((ppd = _ppdOpenFile(ppd_name, _PPD_LOCALIZATION_NONE)) != NULL)
+  if ((ppd = _ppdOpenFile(ppd_name, _PPD_LOCALIZATION_ALL)) != NULL)
```

Comment:
This was actually changed a while back because it caused a huge startup delay.

Reply:
I see; that does make sense. Would using `_PPD_LOCALIZATION_DEFAULT` instead be a significant improvement or almost as bad?

Reply:
@Plombo It's just as bad - any localization slows the loading of the PPD.